internal/plugins/helm/v1/chartutil/chart.go

```diff
@@ -0,0 +1,280 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package chartutil
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/iancoleman/strcase"
+	log "github.com/sirupsen/logrus"
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	"helm.sh/helm/v3/pkg/chartutil"
+	"helm.sh/helm/v3/pkg/cli"
+	"helm.sh/helm/v3/pkg/downloader"
+	"helm.sh/helm/v3/pkg/getter"
+	"helm.sh/helm/v3/pkg/repo"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/kubebuilder/pkg/model/resource"
+)
+
+const (
+
+	// HelmChartsDir is the relative directory within an SDK project where Helm
+	// charts are stored.
+	HelmChartsDir string = "helm-charts"
+
+	// DefaultGroup is the Kubernetes CRD API Group used for fetched
+	// charts when the --group flag is not specified
+	DefaultGroup string = "charts.operator-sdk"
```

Comment:
Users likely won't want `operator-sdk` in their group names.
```suggestion
	DefaultGroup string = "charts"
```